### PR TITLE
#177 adding new rule to detect LGPLv2+ license

### PR DESCRIPTION
**Pull request by rakeshbalusa**
New rule to detect `LPGL 2.0 or later` or later is added

### CODE REVIEW

File: src/licensedcode/data/rules/gpl-2.0-plus_53.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+licenses:
+    - gpl-2.0-plus
```
Comment (pombredanne) [inline src/licensedcode/data/rules/gpl-2.0-plus_53.yml]: `GNU Library General Public License` is the LGPL 2.0 note the GPL 2.0

File: tests/licensedcode/data/licenses/gpl-2.0-plus_47.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+licenses:
+    - gpl-2.0-plus
```
Comment (pombredanne) [inline tests/licensedcode/data/licenses/gpl-2.0-plus_47.yml]: `GNU Library General Public License` is the LGPL 2.0 not the GPL 2.0

Comment (rakeshbalusa) [inline tests/licensedcode/data/licenses/gpl-2.0-plus_47.yml]: my bad..I'll update everything

**Comment by pombredanne** [comment]
LGTM. Merging.

**Comment by rakeshbalusa** [comment]
@pombredanne  I think we need to change this rule to a template. please check the issue #197 . Do I have to create a new ticket and pull request or is there any other way you want me to use?

**Pull request merged**